### DISCUSSION TITLE
Can't setting phpunit tests execution time limit

### DISCUSSION THREAD

**Discussion by potievdev** [Codeception/Codeception#6185] [Ideas]
PHPUnit provide Risky Tests and developer can annotate test as large | medium | small and setting execution time limits. If test takes more than limited time it failes.

**Comment by DavertMik** [comment]
No we don't have it yet and probably not having this. 
Probably it can be implemented as an Extension.

**Comment by aedart** [comment]
I recently run into a problem, where a test timeout is the only way to ensure that my component is doing what it is supposed. While phpunit itself offers this, it would be nice if codeception also could allow this. Maybe inherit parts of the configuration?
